feature-libs/product-configurator/rulebased/components/price/configurator-price.component.ts

```diff
@@ -54,6 +68,14 @@ export class ConfiguratorPriceComponent {
     return quantity ? this.formula.price?.value !== 0 && quantity >= 1 : false;
   }
 
+  /**
+   * Verifies if 'Selected' should be shown
+   *
+   * @return {boolean} - 'true' if 'Selected' should be shown, otherwise 'false'
+   */
+  displaySelected(): boolean {
+    return this.isPriceLightedUp() && !this.displayFormula();
```

Comment:
The wording `Selected` should be the same colour as price:
![image](https://user-images.githubusercontent.com/61147963/213458367-bbe40ff6-98e8-46c0-b135-3b9520b8f582.png)

Reply:
The same as in Accelerator:
![image](https://user-images.githubusercontent.com/61147963/213458655-18df584e-2438-436a-8a4e-2b92722f2c1d.png)

Reply:
done